src/scrapers/base-isracard-amex.ts

```diff
@@ -253,7 +253,7 @@ function getTransactionExtraDetails(servicesUrl: string, month: Moment, accountI
 async function getExtraScrapTransaction(page: Page, options: ExtendedScraperOptions, month: Moment, accountIndex: number, transaction: Transaction): Promise<Transaction> {
   const dataUrl = getTransactionExtraDetails(options.servicesUrl, month, accountIndex, transaction);
   const data = await fetchGetWithinPage<ScrapedTransactionData>(page, dataUrl);
-  const rawCategory = _.get(data, 'PirteyIska_204Bean.sector');
+  const rawCategory:any = _.get(data, 'PirteyIska_204Bean.sector');
```

Comment:
```suggestion
  const rawCategory: string = _.get(data, 'PirteyIska_204Bean.sector');
```

Reply:
There is an issue with the inferred type from loadash "get"... added a condition for data is null and a relevant type which fixed the type inference